src/components/checkboxFilter/index.tsx

```diff
@@ -53,16 +55,35 @@ function CheckboxFilter<ItemKey extends string>({
             key={`filter_${name}_${item.label}`}
             name={`filter_${name}_${item.label}`}
             label={
-              <chakra.span
-                w="full"
-                display="flex"
-                justifyContent="space-between"
-              >
-                <chakra.span>{item.label}</chakra.span>
-                <chakra.span>
-                  {addThousandSeparatorToNumber(item.facet)}
+              item.image ? (
+                <chakra.div
+                  w="full"
+                  display="flex"
+                  alignItems="center"
+                  justifyContent="space-between"
+                >
+                  <chakra.div width="119px" height="80px">
+                    {item.image}
+                  </chakra.div>
+                  <chakra.span display="flex" flexDirection="column">
+                    <chakra.span>{item.label}</chakra.span>
+                    <chakra.span color="gray.400">
```

Comment:
could you double check with a designer? It feels weird to me that it should have a different color depending on if you have an image or not

Reply:
I followed the design in Figma -> https://www.figma.com/file/WvYYKrx8rxw80fwkzhAQwh/Search-Results-%26-Advanced-Search?type=design&node-id=1174-244070&mode=design&t=ZqZPgpqj74xLHFUb-0

I am wondering to whom shouldI ask if Enrique and Sancho are on holidays?

Reply:
Comparing figma filters, it seems that the designs are not aligned with what we already have in production.
On the color filters (in figma) the facets have another text color and in preprod another. Therefore, I will follow preprod and make consistent the label, removing the `gray.400`

Thank you!